js_basics/task5-3.js

```diff
@@ -0,0 +1,5 @@
+const height = 5
+const radius = 10;
+const x = 2;
```

Comment:
what is x? it's not meaningful name. Either change variable name or use literal 2 directly in Math.pow without defining a variable